test/support/fixtures/accounts_fixtures.ex

```diff
@@ -0,0 +1,22 @@
+defmodule Demo.AccountsFixtures do
```

Comment:
```suggestion
defmodule Demo.AccountsFixtures do
  @moduledoc """
  This module defines test helpers for creating
  entities via the `Demo.Accounts` context.
  """

```
This is the same description that was added in the fixtures PR to phoenix: https://github.com/phoenixframework/phoenix/pull/3829. I added these docs to phx_gen_auth last night as a fix to https://github.com/aaronrenner/phx_gen_auth/issues/24.